_assets/js/print-button.js

```diff
@@ -1,14 +1,18 @@
-// Grab our print button:
-var printButton = document.getElementById('crt-page--printbutton');
-
-// Print function:
-function printPage() {
-    window.print();
+const printButtonFunction = () => {
```

Comment:
What do you think about changing this to "PrintButton" as its a component and function is implied?

Reply:
Sure I'll change that back.